frontend/packages/console-shared/src/hooks/useOperands.ts

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { useMemo } from 'react';
+import { coFetchJSON } from '@console/internal/co-fetch';
+import { K8sResourceCommon } from '@console/internal/module/k8s';
+
+export const useOperands = (
+  operatorName: string,
+  operatorNamespace: string,
+): { loaded: boolean; errorMessage: string; operands: K8sResourceCommon[] } => {
```

Comment:
I would suggest returning a tuple like useK8sWatchResource for consistency:
```suggestion
): [operands: K8sResourceCommon[], loaded: boolean, errorMessage: string] => {
```